lib/rubocop/rspec/description_extractor.rb

```diff
@@ -68,11 +68,7 @@ def documented_constant
         end
 
         def cop_subclass?
-          # YARD superclass resolution is a bit flaky: All classes loaded before
-          # RuboCop::Cop::WorkaroundCop are shown as having RuboCop::Cop as
-          # superclass, while all the following classes are listed as having
-          # RuboCop::Cop::RSpec::Cop as their superclass.
-          COP_CLASS_NAMES.include?(yardoc.superclass.path)
+          yardoc.superclass.path == COP_CLASS_NAME
```

Comment:
We're passing our base cop's path to YARD first, so it detects the superclass properly, and doesn't erroneously report some of them to be `RuboCop::Cop`.